pytorch_lightning/trainer/configuration_validator.py

```diff
@@ -30,12 +30,12 @@ def verify_loop_configurations(self, model: LightningModule):
             model: The model to check the configuration.
 
         """
-        if not self.trainer.testing:
+        if not self.trainer.evaluating:
```

Comment:
I would personally rename this to `evaluation_mode`, testing as it was before was a BOOL. This naming made me think it was a BOOL as well though not really the case here. :)